internal/store/bigtable/reader.go

```diff
@@ -107,9 +107,6 @@ func ReadWithGroupRowList(
 	if len(tables) == 0 {
 		return nil, status.Errorf(codes.NotFound, "Bigtable instance is not specified")
 	}
-	if len(tables) != len(rowListMap) {
```

Comment:
Seems like len(tables) <= len(rowListMap), given line 120?

Reply:
Sorry, what I meant was change that condition to failing if len(tables) > len(rowListMap)?  Since we use the length of tables to index into rowListMap